### PR TITLE
doc: add TSC vice chairperson

### PR DESCRIPTION
**Pull request by mhdawson**
Add TSC vice chairperson as discussed in recent
meeting.

Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

**Comment by mhdawson** [comment]
@nodejs/TSC as discussed in the latest meeting and in https://github.com/nodejs/TSC/issues/1201

**Review by gireeshpunathil — APPROVED** [review]

File: TSC-Charter.md [hunk 1]
```diff
@@ -53,10 +53,11 @@ The TSC may, at its discretion, invite any number of non-voting observers to
...
 
 The TSC shall meet regularly using tools that enable participation by the
-community (e.g. weekly on a Google Hangout On Air, or through any other
+community (e.g. weekly on a Zoom call, or through any other
 appropriate means selected by the TSC). The meeting shall be directed by
-the TSC Chairperson. Responsibility for directing individual meetings may be
-delegated by the TSC Chairperson to any other TSC member. Minutes or an
+the TSC Chairperson or vice Chairperson. Responsibility for directing
+individual meetings may be delegated by the TSC Chairperson or vice
+Chairperson to any other TSC member. Minutes or an
```
Comment (Trott) [inline TSC-Charter.md:60]: I don't think we need this change. It already says "Responsibility for directing individual meetings may be delegated by the TSC Chairperson to any other TSC member." So the TSC can choose to delegate a vice chair, and the chair can choose to delegate to the vice chair for any meeting they can't make. If we add the Vice Chair to the Charter, we're tying our hands that we always need to have a Vice Chair, even if we try this and find out it doesn't work.

**Review by Trott — CHANGES_REQUESTED** [review]
I don't think we need this change and I prefer keeping the Charter as non-specific as possible. If it's permitted by the Charter already (and I think it is), then we don't need to add it to the Charter.

**Comment by Trott** [comment]
> I don't think we need this change and I prefer keeping the Charter as non-specific as possible. If it's permitted by the Charter already (and I think it is), then we don't need to add it to the Charter.

In other words, let's close this and just get right to the business of electing (or selecting) one or more Vice Chairs.

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by mhdawson** [comment]
Ok, if we don't need to document the vice-chair anywhere I'm happy to close.

**Pull request closed without merging**